<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Iconos/Iconos.qrc">
    <normaloff>:/Iconos/logo.png</normaloff>:/Iconos/logo.png</iconset>
  </property>
  <widget class="QWidget" name="widget_central">
   <widget class="QLabel" name="img_fondo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: transparent;
	border: none;
	border-bottom: 1px solid #fff;
	color: #fff;
	padding-bottom: 5px;
}
QLineEdit:focus {
		border-bottom: 1px solid #b5b5b5;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Imagenes/fondoLogin.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPagina">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: transparent;
border: none;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Imagenes/login.PNG</normaloff>../Imagenes/login.PNG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUsuario">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>239</y>
      <width>240</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: transparent;
	border: none;
	border-bottom: 1px solid #fff;
	color: #fff;
	padding-bottom: 5px;
}
QLineEdit:focus {
		border-bottom: 1px solid #b5b5b5;
}


</string>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Usuario</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIngresar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>240</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
border-radius: 5px;
background-color: #004A7C;
color:#fff;
padding-left: 15px;
padding-right: 15px;
height: 40px;
}
QPushButton:hover{
background-color: #1C4C96;
}

</string>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtClave">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>240</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: transparent;
	border: none;
	border-bottom: 1px solid #fff;
	color: #fff;
	padding-bottom: 5px;
}
QLineEdit:focus {
		border-bottom: 1px solid #b5b5b5;
}
</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Contrase√±a</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Iconos/Iconos.qrc"/>
 </resources>
 <connections/>
</ui>
